exotic/api/nested_linear_fitter.py

```diff
@@ -1364,6 +1372,7 @@ def prior_transform(upars):
 
     # TODO extend plot, BIC Values
 
+    # should this be here?
```

Comment:
@pearsonkyle Yes, I read this far down. Is this a placeholder for something else? Should we remove it?